### ISSUE TITLE
I have some errors when starting the server

### ISSUE THREAD

**Issue by mizakko** [TheDoctor0/CSGOMod#157]
Well, when I enter cs 1.6 (offline) and enter the /menu command, I get the different options but they all say "ML_NOTFOUND" and apart from that I can't do anything, can someone help me? I think I installed something wrong...

L 09/02/2022 - 18:22:53: Invalid CVAR pointer
L 09/02/2022 - 18:22:53: [AMXX] Run time error 10 (plugin "csgo_clans.amxx") (native "bind_pcvar_num") - debug not enabled!
L 09/02/2022 - 18:22:53: [AMXX] To enable debug mode, add "debug" after the plugin name in plugins.ini (without quotes).
L 09/02/2022 - 18:22:53: Invalid CVAR pointer
L 09/02/2022 - 18:22:53: [AMXX] Run time error 10 (plugin "csgo_accounts.amxx") (native "bind_pcvar_num") - debug not enabled!
L 09/02/2022 - 18:22:53: [AMXX] To enable debug mode, add "debug" after the plugin name in plugins.ini (without quotes).
L 09/02/2022 - 18:22:57: [MySQL] Invalid handle: 0
L 09/02/2022 - 18:22:57: [AMXX] Run time error 10 (plugin "csgo_accounts.amxx") (native "SQL_FreeHandle") - debug not enabled!
L 09/02/2022 - 18:22:57: [AMXX] To enable debug mode, add "debug" after the plugin name in plugins.ini (without quotes).
L 09/02/2022 - 18:24:16: [MySQL] Invalid handle: 0
L 09/02/2022 - 18:24:16: [AMXX] Run time error 10 (plugin "csgo_clans.amxx") (native "SQL_FreeHandle") - debug not enabled!
L 09/02/2022 - 18:24:16: [AMXX] To enable debug mode, add "debug" after the plugin name in plugins.ini (without quotes).
L 09/02/2022 - 18:36:45: Start of error session.
L 09/02/2022 - 18:36:45: Info (map "csgo_dust2_new") (file "addons/amxmodx/logs/error_20220902.log")
L 09/02/2022 - 18:36:45: Invalid CVAR pointer
L 09/02/2022 - 18:36:45: [AMXX] Run time error 10 (plugin "csgo_clans.amxx") (native "bind_pcvar_num") - debug not enabled!
L 09/02/2022 - 18:36:45: [AMXX] To enable debug mode, add "debug" after the plugin name in plugins.ini (without quotes).
L 09/02/2022 - 18:36:45: Invalid CVAR pointer
L 09/02/2022 - 18:36:45: [AMXX] Run time error 10 (plugin "csgo_accounts.amxx") (native "bind_pcvar_num") - debug not enabled!
L 09/02/2022 - 18:36:45: [AMXX] To enable debug mode, add "debug" after the plugin name in plugins.ini (without quotes).
L 09/02/2022 - 18:36:50: [MySQL] Invalid handle: 0
L 09/02/2022 - 18:36:50: [AMXX] Run time error 10 (plugin "csgo_accounts.amxx") (native "SQL_FreeHandle") - debug not enabled!
L 09/02/2022 - 18:36:50: [AMXX] To enable debug mode, add "debug" after the plugin name in plugins.ini (without quotes).
L 09/02/2022 - 18:37:02: [MySQL] Invalid handle: 0
L 09/02/2022 - 18:37:02: [AMXX] Run time error 10 (plugin "csgo_clans.amxx") (native "SQL_FreeHandle") - debug not enabled!
L 09/02/2022 - 18:37:02: [AMXX] To enable debug mode, add "debug" after the plugin name in plugins.ini (without quotes).
![image](https://user-images.githubusercontent.com/97453585/188244224-12c357a1-2c33-4a7a-8ead-22e61e5282fd.png)
![image](https://user-images.githubusercontent.com/97453585/188244239-f60316fe-2768-43f9-81f6-ef930e0aea02.png)

**Comment by TheDoctor0** [comment]
Hi @mizakko, have you configured the MySQL database credentials inside [csgo_mod.cfg](https://github.com/TheDoctor0/CSGOMod/blob/master/cstrike/addons/amxmodx/configs/csgo_mod.cfg)?

**Comment by mizakko** [comment]
> Hi @mizakko, have you configured the MySQL database credentials inside [csgo_mod.cfg](https://github.com/TheDoctor0/CSGOMod/blob/master/cstrike/addons/amxmodx/configs/csgo_mod.cfg)?
> Because looking at those errors the main plugin `csgo_mod.amxx` is not working for you and this causes other plugins to fail as well as translations not being available.

I haven't bought the MySQL base yet, I swear I have to have the MySQL base for the mod to work?

**Comment by TheDoctor0** [comment]
@mizakko, yes, it is a requirement. That said, there are [plenty of free services](https://www.google.com/search?q=free+mysql+database) that provide it, so you don't really need to buy it right away if you want to test the mod.

**Comment by mizakko** [comment]
> @mizakko, yes, it is a requirement. That said, there are [plenty of free services](https://www.google.com/search?q=free+mysql+database) that provide it, so you don't really need to buy it right away if you want to test the mod.

and do you know any page of a free mysql database? It is to do the tests quickly because I have a time limit xd

**Comment by mizakko** [comment]
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_core.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_clans.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_accounts.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_nades.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_vip_svip.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_server_menu.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_transfer.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_operations.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_molotov.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_smoke.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_zeus.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_runboost.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_inspect.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_buymenu.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_ranks.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_assist_revenge.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_stattrak.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_no_distractions.amxx")
L 09/03/2022 - 09:36:33: [AMXX] Load error 17 (invalid file format or version) (plugin "csgo_physics.amxx")

I was able to connect a database but now when I enter the game directly it crashes when entering the sv LAN and then I go to logs and it gives me this message

**Comment by TheDoctor0** [comment]
@mizakko what version of AMX Mod X do you have? 1.8.3+ is needed, I would advise using [BasePack ](https://github.com/AMXX-pl/BasePack)to make it easier.

**Comment by mizakko** [comment]
> @mizakko what version of AMX Mod X do you have? 1.8.3+ is needed, I would advise using [BasePack ](https://github.com/AMXX-pl/BasePack)to make it easier.

I think I installed 1.8.2 lmao

**Comment by mizakko** [comment]
@TheDoctor0  hey, but the basepack is for linux, i use windows xd

**Comment by TheDoctor0** [comment]
@mizakko there are some things in the core plugin that may not work on Windows, so I would advise you to get a test server on some hosting as in 99.9% of the cases it will be running on Linux.
